rope/contrib/codeassist.py

```diff
@@ -347,15 +350,26 @@ def default_templates():
     return {}
 
 
+def _startswith(s1, s2):
```

Comment:
Maybe better will be some like `string`, `prefix` (`sub`) or `haystack`, `needle`. The idea is don't force the user to explore the implementation. The signature must be speaking.